### DISCUSSION TITLE
휴지통 목록 조회에 대해서 클라이언트에게 작업 요청

### DISCUSSION THREAD

**Discussion by gomarag** [codestates/siphoon-note#110] [General]
### 2. 해당 페이지에서 보여야 할 정보는 크게 5가지로 아래와 같다.
  >1 **작성한 글 목록**
  >2 캘린더에 표시될 작성일자 목록
  >3 화면 우측 상단의 본인 계정 정보
  >4 **휴지통으로 보낸 글 목록**
  >5 본인의 습관 기록 * 3

### 3. 문제는 "휴지통의 글 목록"과 "작성 글 목록"이 `isDelete` 필드를 제외한 모든 필드값이 동일함에도 서버로 각각 요청을 보내고 있다는 점.
 - GET /trashes: 휴지통 목록 전체 조회
 - GET /essays: 글 목록 전체 조회
 ----

> **[제안] 📜To @Leekyeongmi**
>처음 `/essays`로 본인 페이지에 접속할 때 `해당 사용자가 작성한 모든 글 목록`을 서버가 보낸다.
- 클라이언트는 `essayList = [{}, {}, {}, ... {}]`로 받은 글 목록 중 `isDeleted` 필드가 `false`면 화면에 출력
- `isDeleted`가 `true`면 화면에서는 숨기고 휴지통에서 볼 수 있게 한다. 이때 휴지통에서는 `#번째 일기 dd-mm-yy '일기의첫줄'`이 리스트로 나열되도록 한다.

**Comment by Leekyeongmi** [comment]
네네! 확인했습니다! 글 목록 중에 isDeleted 필드가  true/false냐에 따라서 필터링해서 렌더링하겠습니닷!
